src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -7,6 +7,95 @@
+        public JsonArray(System.Collections.Generic.IEnumerable<string> jsonValues) { }
+        public JsonArray(System.Collections.Generic.IEnumerable<System.Text.Json.JsonNode> jsonValues) { }
+        [System.CLSCompliantAttribute(false)]
+        public JsonArray(System.Collections.Generic.IEnumerable<ushort> jsonValues) { }
+        [System.CLSCompliantAttribute(false)]
+        public JsonArray(System.Collections.Generic.IEnumerable<uint> jsonValues) { }
+        [System.CLSCompliantAttribute(false)]
+        public JsonArray(System.Collections.Generic.IEnumerable<ulong> jsonValues) { }
+        public int Count { get { throw null; } }
+        public bool IsReadOnly { get { throw null; } }
+        public System.Text.Json.JsonNode this[int idx] { get { throw null; } set { } }
+        public void Add(bool jsonValue) { }
+        public void Add(byte jsonValue) { }
+        public void Add(double jsonValue) { }
+        public void Add(short jsonValue) { }
+        public void Add(int jsonValue) { }
+        public void Add(long jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(sbyte jsonValue) { }
+        public void Add(float jsonValue) { }
+        public void Add(string jsonValue) { }
+        public void Add(System.Text.Json.JsonNode jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(ushort jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(uint jsonValue) { }
+        [System.CLSCompliantAttribute(false)]
+        public void Add(ulong jsonValue) { }
+        public void Clear() { }
+        public bool Contains(bool jsonValue) { throw null; }
+        public bool Contains(byte jsonValue) { throw null; }
+        public bool Contains(double jsonValue) { throw null; }
+        public bool Contains(short jsonValue) { throw null; }
+        public bool Contains(int jsonValue) { throw null; }
+        public bool Contains(long jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(sbyte jsonValue) { throw null; }
+        public bool Contains(float jsonValue) { throw null; }
+        public bool Contains(string jsonValue) { throw null; }
+        public bool Contains(System.Text.Json.JsonNode jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(ushort jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(uint jsonValue) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool Contains(ulong jsonValue) { throw null; }
+        public void CopyTo(System.Text.Json.JsonNode[] array, int arrayIndex) { }
+        public System.Collections.Generic.IEnumerator<System.Text.Json.JsonNode> GetEnumerator() { throw null; }
+        public int IndexOf(System.Text.Json.JsonNode item) { throw null; }
```

Comment:
Same question as above, what should this return in case of duplicate items?

Reply:
Documentation says it should return the first occurrence of a value.